virttest/qemu_monitor.py

```diff
@@ -2087,8 +2100,17 @@ def balloon(self, size):
 
         :param size: int type values.
         """
-        self.verify_supported_cmd("balloon")
-        self.send_args_cmd("balloon value=%s" % size)
+        cmd = "balloon"
+        qmp_event = "BALLOON_CHANGE"
+        self.verify_supported_cmd(cmd)
+        # Clear the event list of QMP monitors
+        self.clear_event(qmp_event)
+        # Send a balloon monitor command
+        self.send_args_cmd("%s value=%s" % (cmd, size))
+        # Look for BALLOON QMP events
+        if not utils_misc.wait_for(lambda: self.get_event(qmp_event), 5):
+            raise QMPEventError(cmd, qmp_event, self.name)
+        logging.info("%s QMP event received" % qmp_event)
```

Comment:
IMO, it's better to show there is qmp event check in the log,
or users won't know there is event check from log.